README.md

```diff
@@ -79,27 +130,33 @@ aws serverlessrepo put-application-policy \
   --statements Principals=*,Actions=Deploy
 ```
 
-## Install Lambda into AWS
+## Releasing
+
+We have setup our CI to automate a releases.  To kick off the process just create
```

Comment:
"to automate a releases" -> "to automate releases"

Reply:
done